### PR TITLE
bech32 convert command

### PR DESCRIPTION
**Pull request by faddat**
bech32 conversions should be a first class citizen, as it is a very useful tool

### CODE REVIEW

File: cmd/junod/cmd/root.go [hunk 1]
```diff
@@ -108,7 +107,7 @@ func initRootCmd(rootCmd *cobra.Command, encodingConfig params.EncodingConfig) {
 		AddGenesisWasmMsgCmd(app.DefaultNodeHome),
 		tmcli.NewCompletionCmd(rootCmd, true),
 		// testnetCmd(app.ModuleBasics, banktypes.GenesisBalancesIterator{}),
-		debug.Cmd(),
+		ConvertBech32Cmd(),
```
Comment (Reecepbcups) [inline cmd/junod/cmd/root.go:110]: Why is the `junod debug` command removed with this? This can still be added on top of the debug.Cmd()

Comment (faddat) [reply cmd/junod/cmd/root.go:110]: sensible!

**Comment by Reecepbcups** [comment]
Feel its better to not spam/fill up the root cmd with random tools

**Review by Reecepbcups — APPROVED** [review]

**Comment by faddat** [comment]
unsure about this but it is nbd so lfg

**Review by JakeHartnell — APPROVED** [review]
LGTM!

**Pull request merged**